inflammation/models.py

```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
+def s_dev(data):
```

Comment:
Function name could be a little more informative

Reply:
Updated in 4473808558d74d39b9de03d59ad33628a276384c